com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/LuaScannerTest.java

```diff
@@ -73,7 +73,7 @@ public void testScanner() throws Exception {
         assertEquals("foo11", modules.get(20));
         assertEquals("foo12", modules.get(21));
         assertEquals("foo13", modules.get(22));
-        
+
         // test require detection with a trailing comment
         assertValidRequire("require \"foo.bar\" -- some comment", "foo.bar");
         assertValidRequire("require 'foo.bar' -- some comment", "foo.bar");
```

Comment:
Perhaps add some test of the troublesome requires we've had?

Reply:
Yep, good point. I added the most recent example where our old solution failed.